src/main/java/com/epam/spring/homework2/config/Config.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.spring.homework2.config;
+
+import com.epam.spring.homework2.beans.BeanB;
+import com.epam.spring.homework2.beans.BeanC;
+import com.epam.spring.homework2.beans.BeanD;
+import com.epam.spring.homework2.beans.BeanF;
+import org.springframework.context.annotation.*;
+import org.springframework.core.annotation.Order;
+
+@Configuration
+@Import(AnotherConfig.class)
+@PropertySource("classpath:application.properties")
+public class Config {
+
+    @Bean(initMethod = "initMethod", destroyMethod = "destroyMethod")
+    @Order(1)
```

Comment:
**@Order**  annotation has impact on the order of elements in the Collection built using, for example **@Autowired**. To check this, we can change the order in which bean creation methods appears in the configuration file - the order of bean creation will change accordingly. 
Please use **@DependsOn** annotation instead. Thank you!